spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/client/RestTemplateBuilder.java

```diff
@@ -444,42 +447,61 @@ public RestTemplateBuilder additionalCustomizers(
 			Collection<? extends RestTemplateCustomizer> customizers) {
 		Assert.notNull(customizers, "RestTemplateCustomizers must not be null");
 		return new RestTemplateBuilder(this.detectRequestFactory, this.rootUri,
-				this.messageConverters, this.requestFactorySupplier,
-				this.uriTemplateHandler, this.errorHandler, this.basicAuthorization,
-				append(this.restTemplateCustomizers, customizers),
-				this.requestFactoryCustomizers, this.interceptors);
+				this.readTimeout, this.connectTimeout, this.messageConverters,
+				this.requestFactorySupplier, this.uriTemplateHandler, this.errorHandler,
+				this.basicAuthorization,
+				append(this.restTemplateCustomizers, customizers), this.interceptors);
 	}
 
 	/**
 	 * Sets the connect timeout in milliseconds on the underlying
 	 * {@link ClientHttpRequestFactory}.
 	 * @param connectTimeout the connect timeout in milliseconds
 	 * @return a new builder instance.
+	 * @deprecated Deprecated in favor of {@link #setConnectTimeout(Duration)}
```

Comment:
The format we use is `@deprecated since 2.1.0 in favour of {@link #setConnectTimeout(Duration)}`.